nl_NL/LC_MESSAGES/discordbot.po

```diff
@@ -112,8 +112,8 @@ msgstr ""
 
 msgid "USAGE_link"
 msgstr ""
-"   link\n"
-"      Link je Sudomemo account met Discord"
+"   koppel\n"
```

Comment:
Line 115 should be unchanged, the command key doesn't change with localization.

Reply:
Changed it back to "link". My bad.